region/maridia/outer/Fish Tank.json

```diff
@@ -1044,11 +1078,17 @@
       "entranceCondition": {
         "comeInRunning": {
           "speedBooster": true,
-          "minTiles": 0.4375
+          "minTiles": 4.4375
```

Comment:
This long of a runway feels similar in difficulty to a shorter one. I feel like it could be lowered to around 3 tiles and feel just about the same. I'm fine either way though.